### PR TITLE
feat: Remove ApiKey need from Emily Get Limit requests

### PR DESCRIPTION
**Pull request by AshtonStephens**
## Description

Closes: #N/A

## Changes

Removes the need for API keys from the emily get limits endpoint

## Checklist:

- [x] I have performed a self-review of my code
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: emily/handler/src/api/handlers/limits.rs [hunk 1]
```diff
@@ -55,7 +55,6 @@ pub async fn get_limits(context: EmilyContext) -> impl warp::reply::Reply {
         (status = 405, description = "Method not allowed", body = ErrorResponse),
         (status = 500, description = "Internal server error", body = ErrorResponse)
     ),
-    security(("ApiGatewayKey" = []))
```
Comment (Jiloc) [inline emily/handler/src/api/handlers/limits.rs:58]: Are we also removing the limit from the set? Will anyone be able to change the limits?

Comment (AshtonStephens) [reply emily/handler/src/api/handlers/limits.rs:58]: No, this is a mistake, this was meant to be the other setter. Fixing.

Comment (AshtonStephens) [reply emily/handler/src/api/handlers/limits.rs]: Fixed.

**Review by Jiloc — APPROVED** [review]
lgtm

**Pull request merged**